fern/pages/sdks/introduction/what-is-an-sdk.mdx

```diff
@@ -0,0 +1,255 @@
+---
+title: What is an SDK?
+description: Generate robust, flexible, and automated SDKs in multiple programming languages.
+---
+
+{/* An SDK (Software Development Kit) is any platform-specific tools that 
+enables faster software development. Traditionally, that could include 
+anything from a compiler to documentation to code libraries. */}
+
+SDKs (Software Development Kits) are code libraries that provide functionality for interacting with your API. 
+By simplifying the process of making API calls, developers have a faster time-to-integration
+and run into fewer errors. 
+
+### SDKs vs. vanilla code

```

Comment:
What does "vanilla code" mean? It's fair to assume a developer has not heard of this term.